server.js

```diff
@@ -30,8 +30,9 @@ const METRICS = [
   'High visibility tickets',
   'Initiative Release Projection',
   'Release Projection',
-  'Release Epics'
-  // 'Defect raise rate'
+  'Release Epics',
+  'Defect raise rate',
+  'Average cycle time for ticket size'
```

Comment:
Can we not just make the "for ticket size" an option of the regular Average Cycle Time metric?